processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -166,11 +171,18 @@ public boolean isPresenceCheckMethod(ExecutableElement method) {
     @Override
     public String getPropertyName(ExecutableElement getterOrSetterMethod) {
         String methodName = getterOrSetterMethod.getSimpleName().toString();
-        if ( methodName.startsWith( "get" ) || methodName.startsWith( "set" ) ) {
-            return IntrospectorUtils.decapitalize( methodName.substring( 3 ) );
-        }
-        else if ( isFluentSetter( getterOrSetterMethod ) ) {
-            return methodName;
+        if ( isFluentSetter( getterOrSetterMethod ) ) {
```

Comment:
feels a bit weird.. just below here you are asserting whether the method starts with "get".. Perhaps move `isFluentSetter` round or make it just `isFluent`

Reply:
You are right, would only make sense to check for the "get". Will change that